src/Products/PlonePAS/tools/configure.zcml

```diff
@@ -0,0 +1,8 @@
+<configure xmlns="http://namespaces.zope.org/zope">
+
+  <adapter
+      provides="Products.CMFCore.interfaces.IMember"
```

Comment:
Really minor: the provides attribute might be redundant (IIRC @implementer should take care of that)

Reply:
@ale-rt I tried ... but since there are two ``@implementer`` Interfaces the adapter needs the provides attribute. Tested...

Reply:
Wow good to know :)